conda/recipes/libcudf/meta.yaml

```diff
@@ -91,6 +91,8 @@ outputs:
     requirements:
       build:
         - cmake {{ cmake_version }}
+      host:
+        - libarrow {{ libarrow_version }}
```

Comment:
```suggestion
```

Reply:
I reverted this in 55f046a58edab38e54511c952a78ac6bfd32a00e. We do want to use `host:` and rely on `run_exports` here.